k8s/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM ubuntu:22.04 as base
+ARG ARCH="amd64"
```

Comment:
**Todo:** In order to integrate correctly with Docker buildx and cross platform builds, this variable needs to be renamed to `TARGETARCH`. See [here](https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/) for more info.

```suggestion
ARG TARGETARCH="amd64"
```